src/basic/socket-util.c

```diff
@@ -870,6 +870,13 @@ static const char* const ip_tos_table[] = {
 
 DEFINE_STRING_TABLE_LOOKUP_WITH_FALLBACK(ip_tos, int, 0xff);
 
+static const char* const socket_protocol_table[] = {
+        [IPPROTO_UDPLITE] = "udplite",
+        [IPPROTO_SCTP] = "sctp",
+};
```

Comment:
hmm, when we add this as proper api to socket-util.c we should probably make this table complete with all known IPPROTO_xyz definitions